### ISSUE TITLE
a little confusion

### ISSUE THREAD

**Issue by houjingyi233** [compsec-snu/razzer#9]
"As a consequence, LLVM Linux will fail to build the entire kernel binary. It is okay if .bc files for files under interests and built-in.bc for each subdirectory (e.g., drivers/built-in.bc, net/built-in.bc) is built."

LLVM Linux will fail to build the entire kernel binary but merge-mempairs.py runs get_address.py and in get_address.py it requires vmlinux file. After I fail to build the entire kernel binary I can run run-partition-analysis.py successfully but merge-mempairs.py will remind me no vmlinux found.

**Comment by lifeasageek** [comment]
ah... that `vmlinux` is the one built by gcc, meaning that you should generate gcc built kernel before running `get_address.py`. Please follow the instruction at https://github.com/compsec-snu/razzer/blob/master/docs/fuzzing.md to build the kernel. We will add more documentation on this.

Note that `mempair` is generated based on LLVM built kernel, and `mapping` is generated through matching `mempair` and gcc built kernel.

**Comment by houjingyi233** [comment]
Thanks for reply.

**Comment by LittleSec** [comment]
after running `run-partition-analysis.py`, log what i see in terminal seem that successfully run, but i `ls -l kernel/partition/v4.17`(for example v4.17), all result files size are 0 byte(of course excluded .bc file). Does it really run successfully?
besides, when i run merge-mempairs.py, it show that all total files(144) merged len is 0.

**Comment by LittleSec** [comment]
oh, I see. I re-check all issue(included closed) and found that I got the error(warning) like what #13 saying. So I reinstall razzer again and this time I log err. I found that I haven't installed svf successfully(don't know why). And when I have installed successfully, it runs successfully(at least, size of file mapping and mempair are not 0).
`find: ‘/home/razzer/Desktop/razzer/tools/SVF/Release-build/lib’: No such file or directory`
